types/gestalt/index.d.ts

```diff
@@ -1660,6 +1643,7 @@ export interface TextAreaProps {
     tags?: ReadonlyArray<React.ReactElement<TagProps, typeof Tag>> | undefined;
     value?: string | undefined;
     readonly?: boolean;
+    labelDisplay?: 'visible' | 'hidden';
```

Comment:
```suggestion
    labelDisplay?: 'visible' | 'hidden' | undefined;
```
can this is `undefined` in native code? (as per e.g. `type`)